cfme/tests/intelligence/reports/test_reports_manual.py

```diff
@@ -188,3 +187,35 @@ def test_reports_timelines_tab():
             1. Timelines tab must not be visible.
     """
     pass
+
+
+@test_requirements.report
+@pytest.mark.customer_scenario
```

Comment:
Hey @valaparthvi 
I can see this file belongs to **reports** test cases only. Although all test cases has this ```@test_requirements.report```. Can you please define it globally.

Reply:
I think it is better to keep it this way, in case I move it to a different file where test_requirements is not `report` and I forget to add a requirement, this would be helpful.